### ISSUE TITLE
okex3 fetchOpenOrders(symbol) ERROR 6247 nodejs.TypeError: this[method] is not a function

### ISSUE THREAD

**Issue by JerroldLee** [ccxt/ccxt#5637]
- OS: MAC
- Programming Language version: NodeJs v11.5.0
- CCXT version: 1.18.1012
- Exchange: okex
- Method: fetchOpenOrders

ERROR 6247 nodejs.TypeError: this[method] is not a function
    at okex3.fetchOrdersByState (/node_modules/ccxt/js/okex3.js:1881:45)

**Comment by kroitor** [comment]
@JerroldLee can you post the code to reproduce it?

- https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#how-to-submit-an-issue
- https://github.com/ccxt/ccxt/wiki/FAQ#im-trying-to-run-the-code-but-its-not-working-how-do-i-fix-it

**Comment by JerroldLee** [comment]
@kroitor Does I not taking it in a right way?Is there any documents?

**Comment by kroitor** [comment]
> Does I not taking it in a right way?

You forgot to post the code to reproduce the issue. Anyways, I did reproduce it on my side and i've uploaded the fix, it will arrive in CCXT 1.18.1013 in 15 minutes. Please update your version in 15 minutes and let us know if you still have the issue or if it is resolved for you.

> Is there any documents?

- https://github.com/ccxt/ccxt/wiki/Manual
- https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#how-to-submit-an-issue
- https://github.com/ccxt/ccxt/wiki/FAQ#im-trying-to-run-the-code-but-its-not-working-how-do-i-fix-it

**Comment by JerroldLee** [comment]
I update to 1.18.1013, but it throws another error now:
ERROR 6874 nodejs.AuthenticationError: okex3 requires `password` credential
    at Object.keys.forEach (/node_modules/ccxt/js/base/Exchange.js:351:27)
    at Array.forEach (<anonymous>)
    at okex3.checkRequiredCredentials (/node_modules/ccxt/js/base/Exchange.js:348:48)
    at okex3.sign (/node_modules/ccxt/js/okex3.js:2648:18)
    at okex3.fetch2 (/node_modules/ccxt/js/base/Exchange.js:522:30)
    at okex3.request (/node_modules/ccxt/js/base/Exchange.js:527:21)
    at okex3.partial (/node_modules/ccxt/js/base/Exchange.js:476:68)
    at okex3.fetchOrdersByState (/node_modules/ccxt/js/okex3.js:1877:45)
    at process.internalTickCallback (internal/process/next_tick.js:77:7)
constructor: undefined
name: "AuthenticationError"

**Comment by JerroldLee** [comment]
@kroitor

**Comment by kroitor** [comment]
@JerroldLee 

> ERROR 6874 nodejs.AuthenticationError: okex3 requires password credential

↑ Is anything not clear with that?

https://github.com/ccxt/ccxt/wiki/Manual#api-keys-setup

**Comment by kroitor** [comment]
@JerroldLee initialize it as follows:

```JavaScript
const exchange = new ccxt.okex3({
    apiKey: 'YOUR_API_KEY',
    secret: 'YOUR_API_SECRET',
    password: 'YOUR_API_PASSWORD', // ←------------- ADD THIS
})
```

**Comment by JerroldLee** [comment]
@kroitor It's ok now,great!

**Comment by kroitor** [comment]
@JerroldLee cool ) thx for reporting!

**Comment by JerroldLee** [comment]
@kroitor There is a similar problem in function of cancelOrder:
 this[method] is not a function
    at okex3.cancelOrder (/node_modules/ccxt/js/okex3.js:1578:45)
    at process.internalTickCallback (internal/process/next_tick.js:77:7)

**Comment by kroitor** [comment]
@JerroldLee will fix it as well, shortly.

**Comment by JerroldLee** [comment]
@kroitor I will wating here for you. ^_^

**Comment by kroitor** [comment]
@JerroldLee this should be fixed in 1.18.1014, let us know if you have further difficulties with it. We will be happy if you report back, whether it solved the issue for you or not.
